app/services/encrypted_attribute.rb

```diff
@@ -22,7 +22,7 @@ def initialize(encrypted, decrypted: nil, cost: nil, user_access_key: nil)
   end
 
   def fingerprint
-    Pii::Fingerprinter.fingerprint(decrypted.downcase)
```

Comment:
this is good -- now that we encrypt more than email, we don't want other attributes treated case insensitively.